embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -353,19 +353,19 @@ private String normalizeTableNameCase(JdbcInputConnection con, String tableName)
     private List<Integer> findIncrementalColumnIndexes(JdbcSchema schema, List<String> incrementalColumns)
         throws SQLException
     {
-        final ArrayList<Integer> builder = new ArrayList<>();
+        final ArrayList<Integer> indices = new ArrayList<>();
         for (String name : incrementalColumns) {
             Optional<Integer> index = schema.findColumn(name);
             if (index.isPresent()) {
```

Comment:
this could be shorten 
```
  indices.add(index.orElseThrow(() -> new ConfigException(String.format(ENGLISH,
          "Column name '%s' is in incremental_columns option does not exist",
          name))));
```

Reply:
Hmm, thought about that, but let's minimize the changes before releasing v0.12.0. Let's try a refactor after that.